codecov.yml

```diff
@@ -6,7 +6,8 @@ coverage:
   range: 70...100
 
   status:
-    project: true
+    project:
+      threshold: 1%
```

Comment:
Maybe this is not needed for this PR? I prefer merging PRs even though they don't have "enough" coverage than getting false positives.